src/Shell/Listener/TinkerEventEmitter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Laravel\Tinker\Shell\Listener;
+
+use Psy\ExecutionLoop\AbstractListener;
+use Psy\Shell;
+
+class TinkerEventEmitter extends AbstractListener
+{
+    public function afterLoop(Shell $shell)
+    {
+        event('tinker.after-loop');
```

Comment:
We can't call a global helper here.

Reply:
The `TinkerEventEmitter` will need a construct with config and event dispatcher contract dependencies.

Reply:
Ok, I'll understand that we need the event dispatcher, but what should the config be used for?

Reply:
You need to call parent::__construct to pass the config to psysh

Reply:
Nvm, I was wrong. That should be done in `TinkerShell` instead.